packages/core/src/middleware/koa-spa-proxy.ts

```diff
@@ -40,7 +41,7 @@ export default function koaSpaProxy<StateT, ContextT extends IRouterParamContext
     // Route has been handled by one of mounted apps
     if (
       !prefix &&
-      Object.values(MountedApps).some((app) => app !== prefix && requestPath.startsWith(`/${app}`))
+      Object.values(mountedApps).some((app) => app !== prefix && requestPath.startsWith(`/${app}`))
```

Comment:
mountedApps.some?

Reply:
i think the code is same as your comment?

Reply:
I mean no need to wrap in a Object.values. Looks like mountedApps is string[] type.

Reply:
ohh right, stupid me